guide/english/algorithms/sieve-of-eratosthenes/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: sieve of Eratosthenes
+---
+
+## sieve of Eratosthenes
+Simple sieve algorithm for finding all prime numbers up to a designated limit.
+The earliest known reference to the sieve attributes it to Eratosthenes of
+Cyrene, a Greek mathematician.
+
+### How it works
+
+The algorithm operates on a set range of numbers starting at 1 through 'n'. It uses the following steps:
+
+  1. Create a list of all numbers in your range.
+  2. Remove 1 from the list, as it is not a prime number.
+  3. Circle the next number on the list.
+  4. Remove all multiples of the most recently circled number from the list.
+  5. Repeat steps 3 & 4 until you no longer have any numbers left in the list.
+
+  The first couple iterations look like this:
```

Comment:
There is a slight confusion on what are you trying to implement here, what is happening is that the use of `*` is making GitHub markdown think that there is a need of a bulleted list, but then there is also a presence of the `1,2,3..` numeric sequence before it.
1. So if you are trying to keep the numerical numbering but want to make the font a little stronger (bolder) then instead of a singular `*`, enclose the headings inside multiple stars `**` like this:- **Sample Heading**
2. If you are wanting a bulleted List then simply remove the numbers preceding the list bullets.